res/controllers/Hercules DJControl Compact.midi.xml

```diff
@@ -12,6 +14,28 @@
         </scriptfiles>
         <controls>
             <control>
+                <group>[EqualizerRack1_[Channel1]_Effect1]</group>
+                <!-- Set the key on the next line to "parameter3" to make the
+                mid knob for Deck A adjust the high eq instead. -->
+                <key>parameter2</key>
+                <status>0xB0</status>
+                <midino>0x3B</midino>
+                <options>
+                    <normal/>
+                </options>
+            </control>
+            <control>
+                <group>[EqualizerRack1_[Channel2]_Effect1]</group>
+                <!-- Set the key on the next line to "parameter3" to make the
```

Comment:
the description tells them to edit the XML, but I don't think that's the place to go into this detail.  This comment is right at the top of the file so if someone does get as far as editing the XML, they'll find it (I hope)